contracts/forge-test/NFTReward_Unit.t.sol

```diff
@@ -22,6 +22,14 @@ import "@jbx-protocol/juice-delegates-registry/src/JBDelegatesRegistry.sol";
 
 import "@jbx-protocol/juice-contracts-v3/contracts/interfaces/IJBOperatable.sol";
 
+import {JBDelegateMetadataHelper} from '@jbx-protocol/juice-delegate-metadata-lib/src/JBDelegateMetadataHelper.sol';
+
+
+bytes4 constant PAY_DELEGATE_ID = bytes4(hex"70");
+bytes4 constant REDEEM_DELEGATE_ID = bytes4(hex"71");
+uint256 constant OVERFLOW = 10e18;
```

Comment:
no please, starting declaring constants outside contracts to avoid scoping them is *not* the way - the ID trick was *only* (as mentioned) a quick fix to not change the constructor args (used, iirc, 196 times)...

Reply:
fair I can push it tomorrow my morning time if that works